chassis/sonic_platform/module.py

```diff
@@ -307,7 +309,19 @@ def set_admin_state(self, up):
         Returns:
             bool: True if the request has been successful, False if not
         """
-        return False
+        if self.get_type() == self.MODULE_TYPE_FABRIC:
+            if up == MODULE_ADMIN_DOWN:
+                logger.log_info("Power off {} module ...".format(self.module_name))
+                nokia_common._power_onoff_SFM(self.hw_slot,False)
+                logger.log_info("Chassis module {} shutdown completed".format(self.module_name))
+            else:
+                logger.log_info("Starting up chassis module {}".format(self.module_name))
+                logger.log_info("Power on {} module ...".format(self.module_name))
```

Comment:
Only one log is needed here. Second log does not provide any extra information. recommend removing second one.